llvm/test/tools/llvm-readobj/ELF/bb-addr-map-pgo-analysis-map.test

```diff
@@ -0,0 +1,221 @@
+## This test checks how we handle PGO Analysis Map in --bb-addr-map.
```

Comment:
nit: I suggest rewording to "how llvm-readobj prints the PGO analysis map with the --bb-addr-map option."